streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/util/ServiceHelper.java

```diff
@@ -29,12 +29,15 @@
 import org.apache.shiro.SecurityUtils;
 import org.apache.shiro.authc.AuthenticationException;
 
+import org.springframework.stereotype.Service;
+
 import java.io.File;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
+@Service
```

Comment:
Can you confirm add @Service?

Reply:
The method of permission authentication has been modified, and there is no need to add the above sections